lua/ranger-nvim.lua

```diff
@@ -206,6 +240,13 @@ function M.open(select_current_file)
 				open_files(SELECTED_FILEPATH, get_open_func())
 			end
 			clean_up()
+
+			-- Close any buffers that were previously pointing to existing files, but don't
+			-- after running ranger. This should close any buffers for files which were
+			-- deleted using ranger.
+            if opts.close_deleted_files then
+                close_empty_buffers(buffers_for_existing_files)
```

Comment:
Since we only need the bufffers for existing files if the option is on, we can merge merge the two if statements to make the code more concise. 

```lua
if opts.close_deleted_files then
  local buffers = get_buffers_for_existing_files()
  close_empty_buffers(buffers)
end
```

Then we can get rid of lines 226-231.

Can you also please fix the indentation here?

Reply:
But the reason that line is before is to get just the buffers which have valid files before running `ranger`, then after `ranger` has closed, looping through those and seeing if they no longer have valid files. I don't think having it in one place would work, would it?

Sorry about the indentation, didn't catch that